app/views/candidate_mailer/_offer_conditions.text.erb

```diff
@@ -1,5 +1,7 @@
-The provider has set the following <%= 'condition'.pluralize(@conditions.count) %>:
+You still need to meet the following <%= 'condition'.pluralize(@conditions.count) %>:
 
-<% @conditions.each do |condition| %>
-  - <%= condition %>
+<% @application_choice.offer.conditions.each do |condition| %>
+<% unless condition.status == 'met' %>
```

Comment:
Can we have a method on condition that checks the status is equal to 'met'?

Reply:
Looking again the code we can benefit from a scope as well:

```ruby
  @application_choice.offer.conditions.not_met.each ...
```

Reply:
Thanks - used enum magic method instead